integration-tests/baas-test-server/cli.ts

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env tsx
+
+/* eslint header/header: 0 */
+
+import chalk from "chalk";
+import dotenv from "dotenv";
+import assert from "node:assert";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import waitFor from "p-wait-for";
+import gha from "@actions/core";
+
+import * as docker from "./docker";
+import * as baasaas from "./baasaas";
+import { UsageError } from "./helpers";
+
+const GITHUB_ACTIONS = process.env.GITHUB_ACTIONS === "true";
+
+// Loading .env from the package directory
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+dotenv.config({
+  path: path.resolve(__dirname, ".env"),
+});
+
+// Loading a .env from cwd too
+dotenv.config();
+
+const COMMON_ID_PREFIX = "arn:aws:ecs:us-east-1:969505754201:task/baas-container-service2/";
+
+function abbreviateId(id: string) {
+  return id.replace(COMMON_ID_PREFIX, "");
+}
+
+function expandId(id: string) {
+  return id.startsWith(COMMON_ID_PREFIX) ? id : COMMON_ID_PREFIX + id;
+}
```

Comment:
@mpobrien for now, abbreviating the id seems safe. Is this just because the service isn't being scaled up?

I should probably check if the if match a pattern instead of starting with a specific prefix here 🤔

Reply:
Either one should work fine, they are both valid in the params to AWS. I would recommend not hardcoding the prefix anywhere in case we ever wanted to change the ECS cluster (which is where the prefix comes from). To guarantee that they will always roundtrip correctly, your best bet is to always just use the IDs exactly as they are returned from BaaSaaS, and don't do post-processing on them at all. If you just want to show an abbreviated ID for human-readable display, splitting on `"/"` and getting the last element works fine.

Reply:
> your best bet is to always just use the IDs exactly as they are returned from BaaSaaS

I'm always sending the exact id back to baasaas, it's mostly for display purposes when running the cli manually.

> splitting on "/" and getting the last element is safe and works fine.

Great - I'll try that instead then 👍